app/views/users/confirmations/invite_admin.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :page_title, "Invite a team member to be an administrator" %>
+
+<%= form_with model: resource, as: resource_name, url: invitation_path(resource_name) do |f| %>
+  <% if @errors %>
```

Comment:
I don't think the `@errors` check is necessary if you instantiate a new User model in the invite_admin method.